Makefile

```diff
@@ -432,34 +420,17 @@ clean-onprem:
 ############
 
 # Current Operator version
-OPERATOR_VERSION ?= 0.0.1
-BUNDLE_OUTPUT_DIR := $(or ${BUNDLE_OUTPUT_DIR},$(BUILD_FOLDER)/bundle)
+OPERATOR_VERSION ?= 0.0.3
+BUNDLE_OUTPUT_DIR ?= deploy/olm-catalog/$(OPERATOR_VERSION)
+BUNDLE_METADATA_OPTS ?= --channels=alpha,ocm-2.3 --default-channel=alpha
 
-# Generate bundle manifests and metadata, then validate generated files.
 .PHONY: operator-bundle
-operator-bundle: create-ocp-manifests
-	set -eux
-	cp ./build/assisted-installer/ocp_role.yaml config/rbac
-	cp ./build/assisted-installer/kube_api_roles.yaml config/rbac
-	cp ./build/assisted-installer/controller_roles.yaml config/rbac
-	cp ./build/assisted-installer/scality-secret.yaml config/assisted-service
-	cp ./build/assisted-installer/scality-public-secret.yaml config/assisted-service
-	cp ./build/assisted-installer/postgres-deployment.yaml config/assisted-service
-	cp ./build/assisted-installer/assisted-installer-sso.yaml config/assisted-service
-	cp ./build/assisted-installer/assisted-service-configmap.yaml config/assisted-service
-	cp ./build/assisted-installer/assisted-service-service.yaml config/assisted-service
-	cp ./build/assisted-installer/assisted-service.yaml config/assisted-service
-	cp ./build/assisted-installer/deploy_ui.yaml config/assisted-service
-	cp ./build/assisted-installer/assisted-installer-local-auth.yaml config/assisted-service
-	# To use --output-dir, needed to break manifests and metadata generation into two steps
-	mkdir -p $(BUNDLE_OUTPUT_DIR)/temp1
-	mkdir -p $(BUNDLE_OUTPUT_DIR)/temp2
-	kustomize build config/manifests | operator-sdk generate bundle --version $(OPERATOR_VERSION) --manifests --output-dir $(BUNDLE_OUTPUT_DIR)/temp1
-	operator-sdk generate bundle --version $(OPERATOR_VERSION) --metadata --input-dir $(BUNDLE_OUTPUT_DIR)/temp1 --output-dir $(BUNDLE_OUTPUT_DIR)/temp2
-	mv $(BUNDLE_OUTPUT_DIR)/temp1/* $(BUNDLE_OUTPUT_DIR)
-	mv $(BUNDLE_OUTPUT_DIR)/temp2/metadata $(BUNDLE_OUTPUT_DIR)
-	rm -rf $(BUNDLE_OUTPUT_DIR)/temp1
-	rm -rf $(BUNDLE_OUTPUT_DIR)/temp2
+operator-bundle:
+	operator-sdk generate kustomize manifests -q
+	cd config/manager && kustomize edit set image controller=$(SERVICE)
+	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(OPERATOR_VERSION) --output-dir $(BUNDLE_OUTPUT_DIR) $(BUNDLE_METADATA_OPTS)
+	# TODO(djzager) structure config/rbac in such a way to avoid need for this
+	rm $(BUNDLE_OUTPUT_DIR)/manifests/assisted-service_v1_serviceaccount.yaml
```

Comment:
It's a small set of commands, but I'm wondering if it would be worth it to put them in a script just to avoid dealing with Make syntax.

Reply:
Now I better understand why `hack/` directories exist :wink: . I personally prefer to keep things in the Makefile to the extent possible. That plus I only see this chunk of steps getting smaller, I think it should stay here.